dask_cudf/io/orc.py

```diff
@@ -0,0 +1,26 @@
+from glob import glob
+
+from dask.base import tokenize
+from dask.compatibility import apply
+import dask.dataframe as dd
+
+import cudf
+
+
+def read_orc(path, **kwargs):
+    filenames = sorted(glob(str(path)))
+    name = "read-orc-" + tokenize(path, **kwargs)
+
+    meta = cudf.read_orc(filenames[0], **kwargs)
+
+    graph = {
+        (name, i): (apply, cudf.read_orc, [fn], kwargs)
+        for i, fn in enumerate(filenames)
+    }
+
+    divisions = [None] * (len(filenames) + 1)
```

Comment:
Also note @randerzander you're not getting any statistics information here, so things like sorted index columns won't come through.  The data will be sorted, but Dask won't know that.

Reply:
Is this because of cuDF GPU ORC reader limitations, or this PR just doesn't load the index information?

Reply:
Both.

More fully featured readers will read through statistics present in a metadata file, and then use that to quickly understand which columns are sorted, and where those values are.  This allows computation systems like Dask dataframe the ability to plan computations like joins-on-sorted-columns much more efficiently. 

Collecting, managging, and planning around this statistics information is something for which we would need both low level code (to read statistics information from binary files) (this would live in cudf or Arrow) and high level code (to arrange computations around those statistics).

Reply:
This PR is a great first step, and we should definitely merge it in.  

Mostly I want to set expectations that we're not promising any level of sophistication around ORC.  This is just a "read in the entire dataset" and compute naively approach.